### ISSUE TITLE
Compatibility with Python 3.9

### ISSUE THREAD

**Issue by honzajavorek** [C4ptainCrunch/ics.py#239]
I'm getting following warnings:

```
/Users/honza/.local/share/virtualenvs/python.cz-s2NxYiyq/lib/python3.7/site-packages/flask_frozen/__init__.py:29
  /Users/honza/.local/share/virtualenvs/python.cz-s2NxYiyq/lib/python3.7/site-packages/flask_frozen/__init__.py:29:
  DeprecationWarning: Using or importing the ABCs from 'collections' instead
  of from 'collections.abc' is deprecated since Python 3.3,and in 3.9 it will stop working
    from collections import Mapping, namedtuple

tests/views_test.py::test_get_involved_cs_renders_ordinary_issue
  /Users/honza/.local/share/virtualenvs/python.cz-s2NxYiyq/lib/python3.7/site-packages/ics/icalendar.py:59:
  DeprecationWarning: Using or importing the ABCs from 'collections' instead
  of from 'collections.abc' is deprecated since Python 3.3,and in 3.9 it will stop working
    elif isinstance(imports, collections.Iterable):
```

**Comment by C4ptainCrunch** [comment]
@honzajavorek Thank you for reporting the issue :)
Could you post here the version of Python and ics.py you are using right when getting the traceback ?

**Comment by N-Coder** [comment]
The first warning is caused by `flask_frozen` and not by `ics`, so its of no concern here. The line referred to in the second warning looks different with the latest ics.py 0.7 (see [here](https://github.com/C4ptainCrunch/ics.py/blob/master/ics/icalendar.py#L59)):
```python
        if imports is not None:
            if isinstance(imports, Container):
```
The [last change](https://github.com/C4ptainCrunch/ics.py/commit/821995e6e60cfce5ada293cecd773e3fb2a717ea#diff-9c14f48c916676959c8ca2df59afd379L65) to that line was made in August 2019, but even before that `collections.abc.Iterable` was used.
So you might be using an outdated version of ics.py.
The warning is probably there for any python version starting from 3.3 (the file paths mentioned in the warning say `.../lib/python3.7/site-packages/...`), but is silenced by default.
It would probably be a good idea to run all CI tests in [python development mode](https://docs.python.org/dev/library/devmode.html), or at least un-silence even minor warnings. 
Additionally, we might want to also already include the latest python 3.9 snapshot in our tests, too catch other problems early. @C4ptainCrunch could you make a PR for this?

**Comment by honzajavorek** [comment]
Thank you, I'm sorry for including the Flask-Frozen warning as well, that was by accident. I'm fixing that in https://github.com/Frozen-Flask/Frozen-Flask/pull/98 already.

The version of ics.py I'm using should be 0.4, according to https://github.com/pyvec/python.cz/blob/082c08006ba9b5ad5d5d407232785ecd0f69e620/Pipfile.lock#L101:L108

**Comment by honzajavorek** [comment]
I did some investigation and I think I'm using this older version because previously there was some sort of conflict between arrow and ics. Now setting the following in my Pipfile allows me to install newer versions of both:

```
ics = "*"
arrow = "*"
```

I need to fix everything in my project now 😅 , but I don't think I see the warning from this issue anymore. I'm sorry for false alarm.

**Comment by N-Coder** [comment]
That was probably fixed by #151. Please note that ICS.py 0.7 is the last version to support arrow up to 0.14. The next version of ICS.py will include some breaking changes, including the removal of arrow, which has a lot of issues handling floating (i.e. timezone-naïve timestamps), see #216.
